docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -112,8 +116,8 @@ When you use the ``roles`` option at the play level, for each role 'x':
 
 - If roles/x/tasks/main.yml exists, Ansible adds the tasks in that file to the play.
```

Comment:
i would rewrite this whole section:

roles look for a main/main.yml/main.yaml by default (you can specify otherwise with include/import_role) in the following directories :

note: vars and default can also match to a directory of the same name and will process all the files contained therin

Reply:
note: meta/ is an exception as it doesn't allow for customization